l10n/en-us.ftl

```diff
@@ -0,0 +1,102 @@
+article-footer_last-modified = This page was last modified on <time data-l10n-name="date">{ $date }</timestamp> by <a data-l10n-name="contributors">MDN contributors</a>.
+article-footer_source_title = Folder: { $folder } (Opens in a new tab)
+
+baseline_asterisk = { $asterisk } Some parts of this feature may have varying levels of support.
+baseline_high_extra = This feature is well established and works across many devices and browser versions. It’s been available across browsers since { $date }.
+baseline_low_extra = Since { $date }, this feature works across the latest devices and browser versions. This feature might not work in older devices or browsers.
+baseline_not_extra = This feature is not Baseline because it does not work in some of the most widely-used browsers.
+baseline_supported_in = Supported in { $browsers }
+baseline_unsupported_in = Not widely supported in { $browsers }
+baseline_supported_and_unsupported_in = Supported in { $supported }, but not widely supported in { $unsupported }
+
+homepage-hero_title = Resources for <u data-l10n-name="developers">Developers</u>,<br> by Developers
+homepage-hero_description = Documenting <a data-l10n-name="css">CSS</a>, <a data-l10n-name="html">HTML</a>, and <a data-l10n-name="js">JavaScript</a>, since 2005.
+
+reference_toc_header = In this article
+
+footer_mofo = Visit <a data-l10n-name="moco">Mozilla Corporation’s</a> not-for-profit parent, the <a data-l10n-name="mofo">Mozilla Foundation</a>.
+footer_copyright = Portions of this content are ©1998–2024 by individual mozilla.org contributors. Content available under <a data-l10n-name="cc">a Creative Commons license</a>.
+
+search_title = Search results for: <em>{ $query }</em>
+search_stats = Found { $results } matches in { $time } milliseconds.
```

Comment:
This string won't work as it is, assuming $results also returns a number of matches, you'll encounter grammatical errors even in English - `Found 1 results in 1 milliseconds.`
The likelihood is higher in a language like Russian which has 3 different plurals depending on the number returned.

Fluent plurals would fix this, but unfortunately it gets too complicated when you have two plural selectors in the same message. To fix this I'd split this into two independent sentences.
```
search-stats-results = { $results ->
    [one] Found { $results } match.
   *[other] Found { $results } matches.
}
search-stats-time = { $time ->
    [one] Search took { $time } millisecond.
    *[other] Search took { $time } milliseconds.
}
```